### DISCUSSION TITLE
Read the Hex file

### DISCUSSION THREAD

**Discussion by karthickrajapetal** [grigorig/stcgal#87] [Q&A]
Hi Sir,
  I need a your help, How to read the HEX file from STC15F104 ,It's very urgent from , Kindly help me Sir.

**Comment by grigorig** [comment]
See the very first point in the FAQ:

https://github.com/grigorig/stcgal/blob/master/doc/FAQ.md#is-it-possible-to-read-code-or-eeprom-memory-out-of-a-chip

**Comment by karthickrajapetal** [comment]
Sir, It's Possible to read the STC15F104, If it's Possible, Kindly give me
the detailed Instructions to read the HEX File, and It's very urgent sir.

*Thanks & Regards,*

M. Karthickraja,

**Comment by karthickrajapetal** [comment]
Sir, Kindly I need a information , we need to read the ADC value in STC15W408AS

**Comment by area-8051** [comment]
See chapter 10 (p. 661) of the documentation: http://www.stcmcudata.com/datasheet/stc/STC-AD-PDF/STC15-English.pdf

**Comment by karthickrajapetal** [comment]
Thank you sir for your information and I need more information , need to read the ROM command in One wire Communication using STC15F.

**Comment by area-8051** [comment]
This is unrelated to STCGAL, and not 8051-specific either.

You have to read the documentation on the one-wire protocol and the data sheet of the device you want to interface with. Additionally, in case an example could help, you might want to have a look at https://github.com/area-8051/uni-STC/tree/main/drivers (see the onewire and DS18B20 sub-folders).
